egs/sre16/v1/local/nnet3/xvector/tuning/run_xvector_1a.sh

```diff
@@ -113,7 +113,7 @@ if [ $stage -le 5 ]; then
   output-layer name=output include-log-softmax=true dim=${num_targets}
 EOF
 
-  steps/nnet3/xconfig_to_configs.py \
+  python steps/nnet3/xconfig_to_configs.py \
```

Comment:
Please change this back to the way it was... there are reasons to rely on what's in the shebang line of the python file.